$NetBSD: patch-src_Makefile,v 1.4 2022/03/13 15:20:13 gdt Exp $

\todo: Figure out why these changes exist and file upstream

--- src/Makefile.orig	2020-10-18 12:56:47.000000000 +0000
+++ src/Makefile
@@ -49,7 +49,7 @@ all:: INSTALL
 
 INSTALL: $(NAME)$(EXEC_EXT)
 # file isn't made for OS X, so check that it's there first
-	(if [ -f $(NAME) ]; then ./$(NAME) -doc install > INSTALLATION; fi)
+##	(if [ -f $(NAME) ]; then ./$(NAME) -doc install > INSTALLATION; fi)
 
 ########################################################################
 ## Miscellaneous developer-only switches
@@ -68,7 +68,7 @@ include Makefile.OCaml
 ######################################################################
 # Installation
 
-INSTALLDIR = $(HOME)/bin/
+INSTALLDIR = ${DESTDIR}$(PREFIX)/bin/
 
 # This has two names because on OSX the file INSTALL shadows the target 'install'!
 install: doinstall
@@ -86,9 +86,7 @@ doinstall: buildexecutable
 	    echo "makefile variable NAME not bound"; \
 	    exit 1 \
 	  ; fi
-	-mv $(INSTALLDIR)/$(NAME)$(EXEC_EXT) /tmp/$(NAME)-$(shell echo $$$$)
 	cp $(NAME)$(EXEC_EXT) $(INSTALLDIR)
-	cp $(NAME)$(EXEC_EXT) $(INSTALLDIR)$(NAME)-$(MAJORVERSION)$(EXEC_EXT)
 
 ######################################################################
 # Demo
